src/Utilities/FNADllMap.cs

```diff
@@ -96,15 +96,10 @@ public static void Init()
 
 			// Get the path to the assembly
 			Assembly assembly = Assembly.GetExecutingAssembly();
-			string assemblyPath = "";
-			if (!String.IsNullOrEmpty(assembly.Location))
-			{
-				assemblyPath = Path.GetDirectoryName(assembly.Location);
-			}
 
 			// Locate the config file
 			string xmlPath = Path.Combine(
-				assemblyPath,
+				AppContext.BaseDirectory,
```

Comment:
That would work only starting from Net 4.6 https://learn.microsoft.com/en-us/dotnet/api/system.appcontext.basedirectory?view=net-7.0

probably should be gated with ifdef

Reply:
This whole file is in an #if NET, so it's .net 5 and newer only here.

Reply:
Then whole change is ok. `AppContext.BaseDirectory` is preferred way to get location of application files and even works with single-file apps.